app/webpack/angularjs/components/art_pieces_browser/art_pieces_browser.scss

```diff
@@ -1,4 +1,5 @@
 @import "mau-mixins";
+@import "gto/gto-mixins";
 
```

Comment:
some day `gto` should get merged in cuz that's where we are.  `mau-mixins` should include *all* mixins